Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Controllers/UsersController.cs

```diff
@@ -39,22 +36,37 @@ public async Task<IActionResult> GetEntry([FromRoute] int id)
             await _db.SaveChangesAsync();
             return Ok(entry);
         }
-        // POST api/values
+        [HttpPost("login")]
+        public ActionResult<Users> Login([FromBody] Users Body)
+        {
```

Comment:
You could rename Body to users and 
` var user = _db.UsersEntries.Where(o => o.Email == Body.Email).FirstOrDefault();`
to **existingUser** 
But again thats your choice

Reply:
Made it more Descriptive

Reply:
Looks beautiful